packages/babel-traverse/src/path/family.ts

```diff
@@ -312,6 +312,46 @@ export function getAllPrevSiblings(this: NodePath): NodePath[] {
   return siblings;
 }
 
+type digit = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9";
+
+// convert "1" to 1 (string index to number index)
+type MaybeToIndex<T extends string> = T extends digit ? number : T;
```

Comment:
Can you try to replace `digit` with
```ts
`${bigint}`
```
?

https://www.typescriptlang.org/play?#code/C4TwDgpgBAKlC8UAGBDAdAEgN4CMCWA5ngHbAC+aOSA3AFC0A2EwUAHgFywJQBE6ALJR50mLAF6c4iPmn4A2ISOZQQk7jIDGi2kA

Inspiration: https://twitter.com/SeaRyanC/status/1420059847156928512

Reply:
Oh! That's brilliant!